selfdrive/car/fw_versions.py

```diff
@@ -25,6 +26,7 @@
 MODEL_TO_BRAND = {c: b for b, e in VERSIONS.items() for c in e}
 REQUESTS = [(brand, config, r) for brand, config in FW_QUERY_CONFIGS.items() for r in config.requests]
 
+CarFw = list[capnp.lib.capnp._DynamicStructBuilder]
```

Comment:
Yes, it should be clear which we're referencing, the list of capnp structs or dicts. I sort of like most types inline so you can see quickly what the type is

Reply:
I've removed it for now